sos/report/plugins/keepalived.py

```diff
@@ -8,10 +8,10 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, RedHatPlugin
+from sos.report.plugins import Plugin, IndependentPlugin
 
 
-class Keepalived(Plugin, RedHatPlugin):
+class Keepalived(Plugin, IndependentPlugin):
```

Comment:
Using `IndependentPlugin` enables this plugin for all distributions. Is the package name the same on all distributions? Along with the file paths being collected?

Reply:
Yes, it looks to be the same everywhere I've checked.